core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -1066,6 +1066,31 @@ public void listStatus(AlluxioURI path, ListStatusContext context,
     return fileInfos;
   }
 
+  /**
+   * {@link DefaultFileSystemMaster} has two private methods associated with sync metadata:
+   * 1. {@link DefaultFileSystemMaster#loadMetadataIfNotExist} sync metadata in load-only mode,
+   * 2. {@link DefaultFileSystemMaster#syncMetadata} sync metadata not in force mode.
+   * Due to loadMetaData need work on both non-load-only mode and force mode, we need custom a
+   * InodeSyncStream object which foreSync param is true and loadOnly param is false.
+   * Further more, synchronize metadata may modify the structure of the inode tree, the lock
+   * pattern of the InodeSyncStream object should be LockPattern.WRITE_EDGE.
```

Comment:
I don't really understand the comment. When you are using words like "foreSync param", "loadOnly param", could you explain them first, so the readers can understand what you are referring to?